generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -192,7 +188,7 @@ module.exports = class extends BaseGenerator {
     appSettings.runtime.key = `adaptive-runtime-${platform}-${integration}`;
     appSettings.runtime.command = {
       [platforms.dotnet]: `dotnet run --project ${botName}.csproj`,
-      [platforms.js]: 'node index.js',
+      [platforms.js]: 'npm run dev --',
```

Comment:
Note: `--` is needed so that subsequent options are passed directly to the underlying command.